manabi/lock.py

```diff
@@ -243,6 +243,10 @@ def __init__(self, refresh: float, postgres_dsn: str):
         self._dict = ManabiPostgresDict(self, self._lock)
 
     def connect(self):
+        try:
+            self._connection.close()
```

Comment:
Might need a comment here to un-confuse future readers. Specifically, that the `AttributeError` from not having `self._connection` at all on first call is also "ok" here
```suggestion
            # Try to close existing connection if there is one. 
            # On first call, this causes a AttributeError, which we can ignore as well
            self._connection.close()
```